src/azul/service/manifest_service.py

```diff
@@ -916,8 +916,7 @@ def manifest_content_hash(self) -> int:
 
     def file_name(self, object_key, base_name: Optional[str] = None) -> str:
         if base_name:
-            file_name_prefix = unicodedata.normalize('NFKD', base_name)
-            file_name_prefix = re.sub(r'[^\w ,.@%&\-_()\\[\]/{}]', '_', file_name_prefix).strip()
```

Comment:
The file name still needs to be sanitized like this. Only the tag value must be base64-encoded.